include/view.h

```diff
@@ -68,7 +68,7 @@ class view {
 		std::string select_filter(const std::vector<filter_name_expr_pair>& filters);
 
 		bool open_in_browser(const std::string& url);
-		void open_in_pager(const std::string& filename);
+		bool open_in_pager(const std::string& filename);
```

Comment:
How about adding a [`warn_unused_result` attribute](https://gcc.gnu.org/onlinedocs/gcc/Common-Function-Attributes.html#index-warn_005funused_005fresult-function-attribute) here and for `open_in_browser` above?